src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/PackageManagerControlLeftSideMaxWidthConverter.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+
+namespace NuGet.PackageManagement.UI
+{
+    public class PackageManagerControlLeftSideMaxWidthConverter : IMultiValueConverter
+    {
+        public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (values[0] == DependencyProperty.UnsetValue || values[1] == DependencyProperty.UnsetValue)
```

Comment:
what's going on here?

Reply:
The converter seemed to be called once before the properties are set while I was testing on how to add a `MultiBinding`. I added this as a safety measure to ensure the converted does not crash in that case. Do you think I should just remove this?

Reply:
not sure why this would be called before they are set, but we definitely don't want it to crash, so keep it this way. 
is there a chance that values[0] can be smaller than values[1] ?

Reply:
`values[0]` is supposed to be the actual width of `_root` which has a minWidth of `LeftSideMinWidth + GridSplitterWidth + RightSideMinWidth` and `values[0]` is constant to `GridSplitterWidth + RightSideMinWidth`.... therefore this converter should return `LeftSideMinWidth` as the smallest value.